tests/stests/manifests/apply_manifests.robot

```diff
@@ -177,3 +179,20 @@ Test Ankaios apply workload specifications via Ankaios Manifest content through
     And the workload "nginx_from_manifest1" shall not exist within "20" seconds
     [Teardown]    Clean up Ankaios
 
+# [stest->swdd~cli-apply-send-update-state~1]
+Test Ankaios apply workload specifications in Ankaios manifest with templated fields
+    [Setup]           Run Keywords    Setup Ankaios
+
+    # Preconditions
+    # This test assumes that all containers in Podman have been created with this test -> clean it up first
+    Given Podman has deleted all existing containers
+    And Ankaios server is started without config
```

Comment:
Just a note, this command is currently unstable as it does not wait until the server is working and can lead to race conditions with the `ank` command later. We don't need to fix this here, but let's keep it in mind in case test start to sporadically fail.

Reply:
Right, understood. This means we fix that in a separate issue to properly wait for the startup of Ankaios components, etc...